### PR TITLE
Add missing horizontal in ListGroupProps

### LINKED ISSUE #4862: Missing Property `horizontal` in ListGroup.d.ts
## Describe the bug

ListGroup with property `horizontal` throws exception in TypeScript.
Seems like property `horizontal` is missing in `ListGroup.d.ts` properties.

## To Reproduce

1. `npx create-react-app app --typescript`
2. `npm install react-bootstrap bootstrap`
3. Add ListGroup in App.js

```javascript
import React from 'react';
import ListGroup from 'react-bootstrap/ListGroup';

import 'bootstrap/dist/css/bootstrap.min.css';

const App: React.FC = () => {
  return (
    <div className="App">
      <ListGroup horizontal>
        <ListGroup.Item>This</ListGroup.Item>
        <ListGroup.Item>ListGroup</ListGroup.Item>
        <ListGroup.Item>renders</ListGroup.Item>
        <ListGroup.Item>horizontally!</ListGroup.Item>
      </ListGroup>
    </div>
  );
}

export default App;
```

```
No overload matches this call.
  Overload 1 of 2, '(props: Readonly<ReplaceProps<"div", BsPrefixProps<"div"> & ListGroupProps>>): ListGroup<"div">', gave the following error.
    Type '{ children: Element[]; horizontal: true; }' is not assignable to type 'IntrinsicAttributes & IntrinsicClassAttributes<ListGroup<"div">> & Readonly<ReplaceProps<"div", BsPrefixProps<"div"> & ListGroupProps>> & Readonly<...>'.
      Property 'horizontal' does not exist on type 'IntrinsicAttributes & IntrinsicClassAttributes<ListGroup<"div">> & Readonly<ReplaceProps<"div", BsPrefixProps<"div"> & ListGroupProps>> & Readonly<...>'.
  Overload 2 of 2, '(props: ReplaceProps<"div", BsPrefixProps<"div"> & ListGroupProps>, context?: any): ListGroup<"div">', gave the following error.
    Type '{ children: Element[]; horizontal: true; }' is not assignable to type 'IntrinsicAttributes & IntrinsicClassAttributes<ListGroup<"div">> & Readonly<ReplaceProps<"div", BsPrefixProps<"div"> & ListGroupProps>> & Readonly<...>'.
      Property 'horizontal' does not exist on type 'IntrinsicAttributes & IntrinsicClassAttributes<ListGroup<"div">> & Readonly<ReplaceProps<"div", BsPrefixProps<"div"> & ListGroupProps>> & Readonly<...>'.  TS2769
```

## Expected behavior

Property `horizontal` should be usable ;-)

## Environment

- Operating System: macOS
- Browser: Chrome 79
- React-Bootstrap Version: 1.0.0-beta.16

### PR DESCRIPTION
**Pull request by mustangore**
This pull request fixes #4862

### CODE REVIEW

File: types/components/ListGroup.d.ts [hunk 1]
```diff
@@ -7,6 +7,7 @@ import { BsPrefixComponent, SelectCallback } from './helpers';
 export interface ListGroupProps {
   variant?: 'flush';
   activeKey?: unknown;
+  horizontal?: unknown;
```
Comment (taion) [inline types/components/ListGroup.d.ts]:
```suggestion
```

File: types/components/ListGroup.d.ts [hunk 2]
```diff
@@ -7,6 +7,7 @@ import { BsPrefixComponent, SelectCallback } from './helpers';
 export interface ListGroupProps {
   variant?: 'flush';
```
Comment (taion) [inline types/components/ListGroup.d.ts]:
```suggestion
   variant?: 'flush';
   horizontal?: boolean;
```

**Comment by mustangore** [comment]
@taion I changed the order.

**Comment by taion** [comment]
ok, but it should still be a `boolean`, no?

**Comment by mustangore** [comment]
You're right, thanks, I overlooked it.
I adjusted it.

**Comment by sjbarag** [comment]
(bumping) What's the current status of this PR?  I've run into #4862 as well and would love to have a fix for it upstream :smiley:

**Comment by taion** [comment]
need another approval to merge; will do so as soon as i get it

**Pull request merged**